src/host/os_getWindowsRegistry.c

```diff
@@ -15,7 +15,7 @@ typedef struct RegKeyInfo
 	char * value;
 } RegKeyInfo;
 
-static HKEY get_key(const char **path)
+HKEY get_registry_key(const char **path)
```

Comment:
Nitpick, but for consistency can you make this `getRegistryKey` (since you're changing anyway)?